code/game/objects/items/plushes.dm

```diff
@@ -698,3 +698,41 @@
 	name = "apocalypse bird plushie"
 	desc = "A large plushie that resembles the beast of the black forest."
 	icon_state = "apocbird"
+
+
+// APR
+/obj/item/toy/plush/benjamingay
```

Comment:
Do you think we can make this a subtype of benjamin so we dont have to define the desc and name. Or would that break things.